src/routes/adminOrder.router.ts

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import { AdminOrderController } from '../controllers';
+import { validateOrderData } from '../middleware';
+
+const router = express.Router();
+
+// 缺少驗證token的middleware
+
```

Comment:
I add verify method and can get `req.user.role` to identify is the coming user a customer or merchant